src/shared/sleep-config.c

```diff
@@ -102,6 +105,180 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         return 0;
 }
 
+/* If battery percentage capacity is less than equal to 5% return success */
+int battery_is_low(void) {
+        int r;
+
+        r = read_battery_capacity_percentage();
+        if (r == -ENOENT)
+               return 0;
```

Comment:
This function returns boolean or negative errno. Hence, please use `false`, instead of `0`.

Reply:
will it not be an implicit cast then?
returning false and then comparing with 0
while (battery_is_low() == 0)
https://github.com/cerebro1/systemd/blob/week1/src/sleep/sleep.c#L272